### PR TITLE
[hal] rename present_nosemaphores

### PR DESCRIPTION
**Pull request by Michael-Lfx**
Fixes #issue
PR checklist:
- [x] `make` succeeds (on *nix)
- [ ] `make reftests` succeeds
- [x] tested examples with the following backends:
  - [x] metal
- [x] `rustfmt` run on changed code

### CODE REVIEW

File: src/hal/src/window.rs [hunk 1]
```diff
@@ -470,7 +470,7 @@ pub trait Swapchain<B: Backend>: fmt::Debug + Any + Send + Sync {
     }
 
     /// Present one acquired image without any semaphore synchronization.
-    unsafe fn present_nosemaphores<'a, C>(
+    unsafe fn present_no_semaphores<'a, C>(
```
Comment (kvark) [inline src/hal/src/window.rs]: we should go further and call it `present_without_semaphores`.
Also `submit_nosemaphores` needs to be updated as well

**Comment by Michael-Lfx** [comment]
Okay. Let me rename it.

File: src/hal/src/window.rs (hunk 1 above)
Comment (Michael-Lfx) [reply src/hal/src/window.rs]: Done. r? cc @kvark

Comment (grovesNL) [reply src/hal/src/window.rs]: @Michael-Lfx `quad/main.rs` still uses `submit_nosemaphores`

Comment (Michael-Lfx) [reply src/hal/src/window.rs]: Fixed 10 minutes ago.

Comment (grovesNL) [reply src/hal/src/window.rs]: Sorry it looks like there is one more (at least) `submit_nosemaphores` in `compute/main.rs`

Comment (Michael-Lfx) [reply src/hal/src/window.rs]: https://github.com/gfx-rs/gfx/commit/71fe6b5c0f94b3c530c93532205efb678ee58be5 should work

**Pull request merged**